webui/src/lib/api/index.js

```diff
@@ -26,6 +24,11 @@ class LocalCache {
 
 const cache = new LocalCache();
 
+export const qs = (queryParts) => {
+    const parts = Object.keys(queryParts).map(key => [key, queryParts[key]]);
```

Comment:
Can we use [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) here, and trust our transpiler to do the right thing for all browsers?
```suggestion
    const parts = Object.entries(queryParts);
```

Reply:
Don't think we have one. But we already use `Object.entries` in our code and the global adoption for this API is on 97.26%.
So I'll take it, thanks!